cpg_workflows/metamist.py

```diff
@@ -151,38 +125,6 @@ def parse(name: str) -> 'AnalysisStatus':
         return {v.value: v for v in AnalysisStatus}[name.lower()]
 
 
-class AnalysisType(Enum):
```

Comment:
You don't want to keep this definition - make it a clear process for how people define new analysis types in production pipelines?

Reply:
I don't know if this makes sense in its current form - this enumeration would track something in code that can be changed with an API call. If we add new entries using the API, we'd have to create a corresponding change to this Enum. 

Upon startup, an API call to find the currently existing analysis types for the given metamist instance could be used so that we don't try and post/create an analysis type that's unacceptable. That could take the place of this Enum so that the interface between the two wouldn't require code changes to remain in sync. Just spitballin' here